doc/ref_impl/cntt-ri/chapters/chapter06.md

```diff
@@ -61,241 +63,400 @@ Thanks to the descriptor file, the NFVi infrastructure deployment could be compl
+
+Only one instance per resource pool.
 
 | Field # | type | mandatory | Instruction |
 |----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| area_name | String | Yes | data center area name |
-| area_center_name | String | Yes | data center name, may compliance with a naming rule.  |
-| room_name | String | Yes | rome static data, helpful in locating issue occurred. |
-| city | String | Yes |  |
-| resource_pool_name | String | Yes |  |
-| outband_network_segment | String | Yes |  |
-| server_templates | List | Yes | server template list included in the resource pool |
-| server_infos | List | Yes | server information list included in the resource pool |
+| RES_POOL_NAME | String | Yes | This is the unique name of the resource pool, could be refered by other parameters |
+| RES_POOL_TYPE | String | Yes | User defined value to identify different hardware or software configuration requirements. |
+
+<p align="center"><b>Table 6-3-1:</b> Resource Pool Information.</p>
 
-<p align="center"><b>Table 6-3-1:</b> Hardware resource description.</p>
 
 <a name="6.3.2"></a>
-### 6.3.2 Server template
-The server template describes the capability of the host like processor, memory, harddrive, raid, NIC/NIC binding, manufacturer, model etc.
-Server template would be assigned to multiple servers, i.e physical hosts.
+### 6.3.2 Global Settings
+The Global settings are provided by the user, contains data like like IP_Type, VLAN_Type, etc.
+
+Only one instance per resource pool.
 
 | Field # | type | mandatory | Instruction |
 |----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| template_name | String | Yes | template name will referenced by dedicated server |
-| manufacturer | String | Yes |  |
-| model | String | Yes |  |
-| processor | String | Yes |  |
-| memory | String | Yes |  |
-| hard_drive | String | Yes |  |
-| raid | String | Yes |  |
-| network_card_infos | List | No | interface list definition|
-| network_card_bond_infos | List | No | NIC bonding, might not be always the case.|
+| IP_TYPE | String | Yes | IPV4 or IPV6 |
+| NETWORK_TYPE | String | Yes | VLAN or VXLAN  |
+| TIMEZONE | String | Yes | The timezone where VIM deployed, like UTC+8  |
+| STORAGE_TYPE | String | Yes |  The Storage type, for example "ceph" |
+| HUGEPAGE_ENABLE | String | Yes | TRUE or FALSE  |
+| HUGEPAGE_SIZE | String | Yes | The storage size that hypervisor set for VM, for example "1GB" |
+| QINQ_ENABLE | String | Yes | TRUE or FALSE |
+| HYPERVISOR_CORES | String | Yes | number of CPU bounded for Hypervisor |
```

Comment:
"number of CPU bounded for Hypervisor" ... number of vCPU (CPU cores or hardware threads) assigned to the Hypervisor?

Reply:
Thanks ! updated